client/landing/stepper/declarative-flow/internals/steps-repository/sites-checker/styles.scss

```diff
@@ -0,0 +1,114 @@
+/* stylelint-disable-next-line scss/at-import-no-partial-leading-underscore */
```

Comment:
Do we need all these CSS?

Reply:
I created https://github.com/Automattic/dotcom-forge/issues/4491 as a follow-up